src/utils/payloadStringify.js

```diff
@@ -43,13 +43,13 @@ export const whitelistKeysRegexes = () => {
   return keyToRegexes([], null, LUMIGO_WHITELIST_KEYS_REGEXES);
 };
 
-export const prune = (str, maxLength) => {
-  let toPrune = str;
-  if (!isString(toPrune)) {
-    logger.warn('Prune was called on a non-string object', toPrune);
-    toPrune = '';
+export const truncate = (str, maxLength, truncationString = '') => {
```

Comment:
this is a fix for a bug discovered after a test broke intermittently when we upgraded the jest dependencies